index.php

```diff
@@ -0,0 +1,54 @@
+<head>
+<link rel="stylesheet" type="text/css" href="normalize.css">
+<link rel="stylesheet" type="text/css" href="style.css">
+<link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
```

Comment:
No indentation here. You might consider indenting the contents of your head tag to make your code more readable